common/db/db.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	mapset "github.com/deckarep/golang-set/v2"
```

Comment:
What is this syntax? Package alias?

Reply:
Yes. `mapset` is the normal alias for this package.